### PR TITLE
Fix #4238: Add missing README for IT tests that don't run by default

### LINKED ISSUE #4238: Some integration tests don't run by default, but have no docs to explain how to run them
I had a hard time finding how to run the `reactive-pg-client` integration test, because tests are skipped by default. Turns out I needed the `-Dtest-postgresql` flag. Some tests have a `README.md` but not most:

```
integration-tests/amazon-dynamodb/README.md
  integration-tests/jpa-mariadb/README.md
  integration-tests/jpa-mssql/README.md
  integration-tests/jpa-postgresql/README.md
  integration-tests/jpa/README.adoc
  integration-tests/keycloak/README.md
  integration-tests/neo4j/README.md
```

The following seem to have `<skip>true` in their build, so might be disabled by default too (need to check) and have no `README`:

- integration-tests/hibernate-search-elasticsearch
- integration-tests/infinispan-client
- integration-tests/mongodb-client
- integration-tests/reactive-mysql-client
- integration-tests/reactive-pg-client

### PR DESCRIPTION
**Pull request by FroMage**

### CODE REVIEW

File: integration-tests/reactive-mysql-client/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Reactive MySQL example
+
+## Running the tests
+
+By default, the tests of this module are disabled.
+
+To run the tests in a standard JVM with MySQL started as a Docker container, you can run the following command:
```
Comment (gsmet) [inline integration-tests/reactive-mysql-client/README.md]:
```suggestion
To run the tests in a standard JVM with MariaDB started as a Docker container, you can run the following command:
```

File: integration-tests/reactive-mysql-client/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+mvn clean install -Dtest-mariadb -Ddocker
+```
+
+Additionally, you can generate a native image and run the tests for this native image by adding `-Dnative`:
+
+```
+mvn clean install -Dtest-mariadb -Ddocker -Dnative
+```
+
+If you don't want to run MySQL as a Docker container, you can start your own MySQL server. It needs to listen on the default port and have a database called `hibernate_orm_test` accessible to the user `hibernate_orm_test` with the password `hibernate_orm_test`.
```
Comment (gsmet) [inline integration-tests/reactive-mysql-client/README.md]:
```suggestion
If you don't want to run MariaDB/MySQL as a Docker container, you can start your own MariaDB/MySQL server. It needs to listen on the default port and have a database called `hibernate_orm_test` accessible to the user `hibernate_orm_test` with the password `hibernate_orm_test`.
```

File: integration-tests/hibernate-search-elasticsearch/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Hibernate Search example with ElasticSearch
```
Comment (gsmet) [inline integration-tests/hibernate-search-elasticsearch/README.md]:
```suggestion
# Hibernate Search example with Elasticsearch
```

File: integration-tests/hibernate-search-elasticsearch/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+## Running the tests
+
+By default, the tests of this module are disabled.
+
+To run the tests in a standard JVM with ElasticSearch started in the JVM, you can run the following command:
```
Comment (gsmet) [inline integration-tests/hibernate-search-elasticsearch/README.md]:
```suggestion
To run the tests in a standard JVM with Elasticsearch started in the JVM, you can run the following command:
```

**Review by kenfinnigan — APPROVED** [review]

**Review by gsmet — COMMENTED** [review]
@kenfinnigan well, I was posting this review... so I will create a new PR then :).

**Comment by kenfinnigan** [comment]
Sorry @gsmet

**Comment by gsmet** [comment]
Just bad timing :).

**Pull request merged**